x-pack/plugins/siem/public/components/event_details/columns.tsx

```diff
@@ -63,29 +67,52 @@ export const getColumns = (id: string) => [
     name: i18n.VALUE,
     sortable: true,
     truncateText: false,
-    render: (values: ItemValues[]) => (
+    render: (values: ToStringArray | null | undefined, data: DetailItem) => (
       <EuiFlexGroup direction="column" alignItems="flexStart" component="span" gutterSize="none">
-        {values.map(item => (
-          <EuiFlexItem grow={false} component="span" key={`${id}-value-${item.valueAsString}`}>
-            <WithHoverActions
-              hoverContent={
-                <HoverActionsContainer data-test-subj="hover-actions-container">
-                  <EuiToolTip content={i18n.COPY_TO_CLIPBOARD}>
-                    <WithCopyToClipboard text={item.valueAsString} titleSummary={i18n.VALUE} />
-                  </EuiToolTip>
-                </HoverActionsContainer>
-              }
-              render={() => item.value}
-            />
-          </EuiFlexItem>
-        ))}
+        {values != null &&
+          values.map((value, i) => (
+            <EuiFlexItem
+              grow={false}
+              component="span"
+              key={`${eventId}-${data.field}-${i}-${value}`}
+            >
+              <WithHoverActions
+                hoverContent={
+                  <HoverActionsContainer data-test-subj="hover-actions-container">
+                    <EuiToolTip content={i18n.COPY_TO_CLIPBOARD}>
+                      <WithCopyToClipboard text={value} titleSummary={i18n.VALUE} />
+                    </EuiToolTip>
+                  </HoverActionsContainer>
+                }
+                render={() => (
+                  <DefaultDraggable
+                    data-test-subj="ip"
+                    field={data.field}
+                    id={`event-details-field-value-${eventId}-${data.field}-${i}-${value}`}
+                    tooltipContent={data.field}
+                    value={value}
+                  >
+                    <FormattedFieldValue
+                      contextId={'event-details-field-value'}
+                      eventId={eventId}
+                      fieldName={data.field}
+                      fieldType={data.type}
+                      value={value}
+                    />
+                  </DefaultDraggable>
+                )}
+              />
+            </EuiFlexItem>
+          ))}
       </EuiFlexGroup>
     ),
   },
   {
     field: 'description',
     name: i18n.DESCRIPTION,
-    render: (description: string) => <SelectableText>{description}</SelectableText>,
+    render: (description: string, data: DetailItem) => (
```

Comment:
Is this type correct? Below you have `${description || ''}` which seems to hint that this type can be either a string or null or undefined.